certbot-apache/certbot_apache/assertions.py

```diff
@@ -10,6 +10,10 @@ def assertEqual(first, second):
 
     if isinstance(first, interfaces.CommentNode):
         assertEqualComment(first, second)
+    elif isinstance(first, interfaces.BlockNode):
+        # this check needs to be first, as BlockNodes are also instances of
+        # DirectiveNode
+        assertEqualBlock(first, second)
     elif isinstance(first, interfaces.DirectiveNode):
         assertEqualDirective(first, second)
```

Comment:
do we want it to talk check `assertEqualDirective` for `BlockNodes` then?

Reply:
Yeah, the `assertEqualBlock` had just a single different step; asserting the `BlockNode` interface implementation. I moved this single step to the `assertEqual` itself, and removed the `assertEqualBlock` and the accompanying conditional from here.